apps/android/app/src/main/java/chat/simplex/app/views/chat/ChatItemInfoView.kt

```diff
@@ -80,23 +80,27 @@ fun ChatItemInfoView(ci: ChatItem, ciInfo: ChatItemInfo, devTools: Boolean) {
       if (!sent) {
         InfoRow(stringResource(R.string.info_row_received_at), localTimestamp(ci.meta.createdAt))
       }
-      when (ci.meta.itemDeleted) {
+      when (val itemDeleted = ci.meta.itemDeleted) {
```

Comment:
maybe get timestamp outside with some CIDeleted.itemDeletedTs not to have two ifs and check it's not null?